platform/darwin/src/MGLTileSet.h

```diff
@@ -19,6 +19,27 @@ typedef NS_ENUM(NSUInteger, MGLTileSetScheme) {
 
 /**
  An `NSArray` of `NSString` objects that represent the tile templates.
+ 
+ Tile URL template strings are a loose open standard for referring to custom
+ locations from which to download tiles, whether vector or raster.
+
+ Currently the tokens `x`, `y`, and `z` are supported for tile locations
+ in the world grid, as well as `ratio` for use in raster tiles, which allows
+ pre-rendered imagery to adapt to the current `UIScreen.scale`.
+
+ Examples:
+ 
+ <ul>
+ <li><code>http://abc.com/tiles/{z}/{x}/{y}.pbf</code> might become 
+    <code>http://abc.com/tiles/14/6/9.pbf</code></li>
+ <li><code>http://abc.com/tiles/{z}/{x}/{y}{ratio}.png</code> might become 
+    <code>http://abc.com/tiles/14/6/9@2x.png</code></li>
+ </ul>
+
+ Only the first tile URL template string specified is used.
+
+ @see <a href="https://wiki.openstreetmap.org/wiki/Slippy_map_tilenames">OpenStreetMap's
```

Comment:
While this is an incredibly useful wiki page, we need to treat it with caution because it assumes 256×256-pixel raster tiles as opposed to the 512×512-pixel tiles we use. In other words, `{z}` is one off.